tests/unit/backend/utils/field-validation/date-validation.spec.js

```diff
@@ -462,4 +462,34 @@ describe('Date field validation', () => {
 
     jasmine.clock().uninstall()
   })
+
+  it('should disallow responses submitted for hidden fields', () => {
+    jasmine.clock().install()
```

Comment:
nit: to move jasmine clock install/uninstall to `beforeAll` and `afterAll`

Reply:
edited